addons/udes_sale_stock/models/__init__.py

```diff
@@ -8,3 +8,4 @@
 from . import stock_warehouse
 from . import stock_picking
 from . import stock_picking_type
+from . import courier
```

Comment:
nit: rename this and file to `udes_courier` and `udes_courier.py`

Reply:
Good point, `udes.carrier` and `udes_carrier.py` please

Reply:
(after seeing adams comment, `udes_carrier` and `udes_carrier.py` :smile: